dataplicity/portforward.py

```diff
@@ -222,18 +229,20 @@ def host_port(self):
         """A tuple of (host, port) as a convenience for socket.connect."""
         return (self.host, self.port)
 
-    @property
-    def url(self):
```

Comment:
Why remove this?

Reply:
it was only used in Connection. However with the above comments, I think I'll move it there